api/src/services/users.ts

```diff
@@ -98,6 +99,18 @@ export class UsersService extends ItemsService {
 		}
 	}
 
+	private validateEmails(emails: string[]) {
+		const schema = Joi.string().email().required();
+
+		for (const email of emails) {
+			const { error } = schema.validate(email);
+
+			if (error) {
+				throw new FailedValidationError(joiValidationErrorItemToErrorExtensions(error.details![0]!));
```

Comment:
```suggestion
				throw new FailedValidationError(joiValidationErrorItemToErrorExtensions(error.details[0]!));
```

Also could we maybe adapt `joiValidationErrorItemToErrorExtensions` to accept `ValidationErrorItem | undefined` so we don't have to use the non-null assertion here?

```suggestion
				throw new FailedValidationError(joiValidationErrorItemToErrorExtensions(error.details[0]));
```